airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/streams.py

```diff
@@ -190,8 +190,8 @@ def url_base(self) -> str:
         Docs: https://business-api.tiktok.com/marketing_api/docs?id=1701890920013825
         """
         if self.is_sandbox:
-            return "https://sandbox-ads.tiktok.com/open_api/v1.2/"
-        return "https://business-api.tiktok.com/open_api/v1.2/"
+            return "https://sandbox-ads.tiktok.com/open_api/v1.3/"
```

Comment:
nit: should we make `1.3` a variable so we don't have to update it 4 times every time we upgrade?

Reply:
@girarda it's 2 times, and only 1 place in code. It's fine since there are no other places we should adjust once another  API version comes up, so we're good as is.